app/resources/group_resource.rb

```diff
@@ -6,7 +6,7 @@ class GroupResource < BaseResource
 
   attributes :about, :locale, :members_count, :name, :nsfw, :privacy, :rules,
     :rules_formatted, :leaders_count, :neighbors_count, :featured, :tagline,
-    :last_activity_at
+    :last_activity_at, :pinned_post_id
```

Comment:
You don't need to expose the `pinned_post_id`, it's exposed automatically by `has_one :pinned_post`